src/cli/crawl-specs.js

```diff
@@ -163,7 +163,7 @@ async function crawlSpec(spec, crawlOptions) {
  * @return {Promise<Array(Object)} The promise to get an array of complete
  *   specification descriptions
  */
-async function crawlList(speclist, crawlOptions, resultsPath) {
+async function crawlList(speclist, crawlOptions) {
```

Comment:
Indeed, the parameter is useless. Since you're at it, could you drop the parameter from the function call on line 395?